src/lib/viewers/BaseViewer.js

```diff
@@ -1000,6 +1006,30 @@ class BaseViewer extends EventEmitter {
         return AnnotationMode.NONE;
     }
 
+    /**
+     * Updates localStorage with the state of the ftux cursor toggle
+     *
+     * @protected
+     * @param {string} key
+     * @return {void}
+     */
+    handleFtuxCursorToggle(key) {
```

Comment:
Typically we use `handle*` naming convention for event handlers, maybe in this case it could be `applyCursorFtux` or something along those lines?

Reply:
That sounds good.